modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.d.ts

```diff
@@ -117,6 +117,15 @@ export declare type TAction =
 			};
 			type: TYPES.SET_SELECTED_OBJECT_DEFINITION_NODE;
 	  }
+	| {
+			payload: {
+				draggable: boolean;
```

Comment:
Hey @Artartur, you could rename this propertie to `objectDefinitionNodeDraggable` to be consistency with `SET_SELECTED_OBJECT_DEFINITION_NODE_DRAGGRABLE`

Reply:
done!